src/views/download/download.scss

```diff
@@ -0,0 +1,96 @@
+@import "../../colors";
+@import "../../frameless";
+
+$developer-spot: $splash-blue;
+
+#view {
+    padding: 0;
+}
+
+.download {
+    .title-banner {
+        &.masthead {
+            background-color: $developer-spot;
+            padding-bottom: 0;
+
+            h1 {
+                margin: 0 0 2rem 0;
+                text-align: left;
+                color: $ui-white;
+            }
+
+            p {
+                margin: 0;
+                width: $cols6;
+                text-align: left;
+                color: $ui-white;
+
+                a {
+                    border-bottom: 1px solid $ui-white;
+                    color: $ui-white;
+                }
+            }
+
+            .band {
+                $band-color: #3baddd;
+                
+                margin-top: 2rem;
+                background-color: $band-color;
+                padding: 1rem 0;
+            }
+
+            .sub-nav {
+                text-align: left;
+                justify-content: flex-start;
+
+                li {
+                    margin: 0 .5rem 0 0;
+                }
+            }
+        }
+    }
+    
+    .download-content {
+        padding-bottom: 2rem;
+    }
+    
+    .installation {
+        display: flex;
+        flex-wrap: wrap;
+        align-items: flex-start;
+        justify-content: space-between;
+    }
+        
+    .installation-column {
+        max-width: $cols4;
+    }
+    
+    .installation-downloads {
+        padding-left: 0;
+        list-style: none;
+    }
+    
+    .installation-downloads-item {
+        margin: .25rem;
+        padding: 0;
+    }
+    
+    section {
+        margin-bottom: 2rem;
+    }
+    
+    .mod-link {
+        color: $ui-white;
+    }
+    
+    @media only screen and (max-width: $mobile - 1) {
+        p {
+            width: $cols3;
```

Comment:
This looks a bit awkward for the sections on mobile:
<img width="243" alt="screen shot 2017-05-31 at 9 04 41 am" src="https://cloud.githubusercontent.com/assets/1815772/26633077/361aa258-45e0-11e7-9b1d-a827c995f561.png">
I think there are 2 possible directions for this:
* remove this, leaving the width at 100%
* set the width to 90%, and add `margin: auto;`

Given that the 100% width is in our typography settings, I'm inclined to just remove this setting for now, personally